### PR TITLE
Milestone 2: Math Magician | Calculator skeleton

### PR DESCRIPTION
**Pull request by michael-duke**
## Math Magician, milestone 2 : Calculator Skeleton

In this milestone I have:

- Used React components.
- Developed the UI of the calculator.
- Create a React component `Calculator.js`  that matches the required design, with accordance to my theme color and feel.
- Followed JavaScript best practices
- Used Tailwind CSS and followed HTML/CSS best practices.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
+import '../styles/Calculator.css';
+import React from 'react';
+
+/* eslint-disable-next-line react/prefer-stateless-function */
```
Comment (arslanbisharat) [inline src/components/Calculator.js]: - [ ] Let's make sure you don't disable linter rules as these are here to help us write clean code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @michael-duke, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You have added a descriptive Readme file. Good Job. ✔️
- There is no linter error is present. :x:
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (michael-duke) [reply src/components/Calculator.js]: Hello @arslanbisharat  thank you for your feedback, but as per the requirement of this milestone all components must `class based components`, hence the reason I need to disable the particular lint
Kindly advise on that, if not I will be forced to use `stateless function` which is not according to the project requirement

**Comment by michael-duke** [comment]
I fixed it @arslanbisharat using a constructor method and defining the state for later use. Thank you for your feedback 😄

**Review by victorokoroji — APPROVED** [review]
Hi @michael-duke, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by michael-duke** [comment]
Thank you @vickymarz for your feedback, happy coding 😄 🚀

**Pull request merged**